js/slouchdetection.js

```diff
@@ -36,3 +49,26 @@ browser.runtime.onMessage.addListener((message, sender, sendResponse) => {
         }
     }
 });
+
+function slouchDetect(faces) {
+    let calibratedFaceCenter = {
+        x: m_CameraController.calibrateInfo.face.faceRectangle.left + ((m_CameraController.calibrateInfo.face.faceRectangle.width) / 2),
+        y: m_CameraController.calibrateInfo.face.faceRectangle.top + ((m_CameraController.calibrateInfo.face.faceRectangle.height) / 2)
+    }
+
+    let faceCenter = { //Get center of face in frame
+        x: faces[0].faceRectangle.left + ((faces[0].faceRectangle.width) / 2),
+        y: faces[0].faceRectangle.top + ((faces[0].faceRectangle.height) / 2),
+    }
+
+    if (faces[0].faceRectangle !== null) {
+        if (faces[0].faceRectangle.height > 190 || faces[0].faceRectangle.width > 190) {
```

Comment:
Why are you comparing to these numbers? What do they mean? Should they be constants?